File: app/src/main/java/com/github/kongpf8848/animation/activity/tween/TweenScaleActivity.java
package com.github.kongpf8848.animation.activity.tween;

import android.view.View;

import com.github.kongpf8848.animation.activity.BaseActivity;
import com.github.kongpf8848.animation.R;
import com.github.kongpf8848.animation.activity.tween.scale.CupidActivity;
import com.github.kongpf8848.animation.activity.tween.scale.QimangxingActivity;
import com.github.kongpf8848.animation.activity.tween.scale.WeizhiActivity;

import butterknife.OnClick;


/**
 * Created by jack on 2016/8/5.
 */
public class TweenScaleActivity extends BaseActivity {

    @Override
    protected int getLayoutId() {
        return R.layout.activity_tween_scale;
    }


    @OnClick(R.id.button1)
    public void onButton1(View view) {
        startActivity(CupidActivity.class);
    }

    @OnClick(R.id.button2)
    public void onButton2(View view) {
        startActivity(WeizhiActivity.class);
    }

    @OnClick(R.id.button3)
    public void onButton3(View view) {
        startActivity(QimangxingActivity.class);
    }




}

File: app/src/main/java/com/github/kongpf8848/animation/activity/tween/TweenScaleActivity.kt
package com.github.kongpf8848.animation.activity.tween

import android.view.View
import butterknife.OnClick
import com.github.kongpf8848.animation.R
import com.github.kongpf8848.animation.base.BaseToolbarActivity
import com.github.kongpf8848.animation.activity.tween.scale.CupidActivity
import com.github.kongpf8848.animation.activity.tween.scale.QimangxingActivity
import com.github.kongpf8848.animation.activity.tween.scale.WeizhiActivity

class TweenScaleActivity : BaseToolbarActivity() {

    override fun getLayoutId(): Int {
        return R.layout.activity_tween_scale
    }

    @OnClick(R.id.button1)
    fun onButton1(view: View?) {
        startActivity(CupidActivity::class.java)
    }

    @OnClick(R.id.button2)
    fun onButton2(view: View?) {
        startActivity(WeizhiActivity::class.java)
    }

    @OnClick(R.id.button3)
    fun onButton3(view: View?) {
        startActivity(QimangxingActivity::class.java)
    }
}